api/package.json

```diff
@@ -14,12 +14,12 @@
         "start": "npm run build && node dist/server.js"
     },
     "dependencies": {
-        "express": "^4.21.0",
-        "express-oauth-jwt": "^2.0.2"
```

Comment:
This is our library, so we should update the dependencies there anyway. I hoped more people would see that we have it if we use it also in this project. I think either approach works, however.

Reply:
I will create a maintenance ticket to update the library.

Reply:
good